portfolio/src/main/webapp/script.js

```diff
@@ -127,3 +127,23 @@ const slideToggle = (elementId, duration)  => {
     slideUp(elementId, duration);
   }
 }
+
+/* fetches comments content from webserver and adds to DOM in container with id elementID */
+const addToDOM = (elementId) => {
+  fetch("/comments").then(response => response.json()).then((comments) => {
+
+    // add each parsed json.commentText to the DOM
+    const destinationDiv = document.getElementById(elementId);
```

Comment:
How do you know it's a div? Perhaps `targetElement` or even just `element`

Reply:
Also, why not define this element just once per page load?

Reply:
My b, I thought it was

Reply:
and set it to `destination`